### PR TITLE
feat: basic support for sorting items #7

### PR DESCRIPTION
**Pull request by ner0-m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,31 +74,40 @@ require("cmp_git").setup({
...
             state = "opened", -- opened, closed, locked, merged
+            sort_by = nil, -- nil, "iid", "title", "description", or custom function
         },
     },
 })
 ```
 
+The custom function for sorting, takes the item (issue, pull/merge request or mention) as argument, and should
+return a string, which will be used to fill the `sortText` field.
+
```
Comment (hrsh7th) [inline README.md:110]: I think default sort should be `updated`. What do you think about it?

(I was planning to do it but you are fast than by me. Thank you!)

Comment (ner0-m) [reply README.md:110]: I'll have a look. I'll keep you posted

Comment (ner0-m) [reply README.md:120]: So I've worked a little on it now. It's note the most elegant ting in the world.

Notable is that to make it work, `cmp.sort.comparator.sort_text`´has to be called before `score`. If the overall design is nice, I'll add it to the readme

**Comment by ner0-m** [comment]
I've force pushed the changes such that the commit history is a bit nicer :-) Just heads up

**Comment by ner0-m** [comment]
@petertriho if you don't have anything to add, I'll integrate this once #31 is merged, and merge it as well

**Comment by ner0-m** [comment]
Alright, could anyone test this and check if this works, if so this is ready to be merged

**Pull request merged**